src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -93,4 +93,7 @@
         public static final String SAE_MISSING_OBJECT_CODES_SUB_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE = "SAE_MISSING_OBJECT_CODES_SUB_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE";
     }
 
+    public static final String CASH_ADVANCE_INDICATOR = "CASH ADVANCE";
```

Comment:
This is the hard coded string that will be on a Concur data file line to indicate a data line represents a cash advance and should be further looked at for possible processing and inclusion in the request extract PDP file being created.  
Would prefixing with "REQUEST_EXTRACT_" help to clear things up?